episodes/collaborating_newcomers.md

```diff
@@ -0,0 +1,169 @@
+
+* [`README.md` template](https://github.com/carpentries/lesson-development-training/blob/main/files/templates/README-template.md?plain=1)
+* [`CONTRIBUTING.md` template](https://github.com/carpentries/lesson-development-training/blob/main/files/templates/CONTRIBUTING-template.md?plain=1)
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::: callout
+
+#### Digital Object Identifiers
+We recommend that lesson developers obtain a [Digital Object Identifier (DOI)](https://the-turing-way.netlify.app/communication/citable/citable-steps.html?highlight=doi#dois) as soon as feels appropriate, at the latest when the lesson reaches the beta phase. You can use this DOI in the citation information (e.g. [in the `identifiers` field of your `CITATION.cff`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#identifiers)), to allow people to cite a particular version of your lesson, e.g. a snapshot of the lesson when it entered beta testing, captured as a Zenodo record.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+### Cultivate and present a welcoming project culture
+Many potential contributors will "lurk" on an open source project before making their first contribution and many occasional contributors are probably paying attention to activity on the repository too.
+People often feel intimidated about contributing to a project for the first time and anything you can do to reduce that barrier and encourage them to participate will help you grow the community around your project.
+
+#### Be intentional about your public communications
+All projects have a culture, much of which will be visible on a public GitHub repository, whether maintainers try to actively establish and nurture it or not.
+Investing time and energy in communicating in a polite and friendly way, even among a small group of team members who collaborate with each other all the time, shows newcomers how they can expect to be received when they make their first contribution.
+Modeling respectful behaviour between maintainers demonstrates to anybody watching (or looking back at past discussion threads) something important about the culture of your project.
+
+Discuss with your team members how you want to work together then establish those norms and work to maintain them.
+
+#### Be responsive
+It helps to process and respond to contributions quickly, especially when somebody is participating in the project for the first time.
+Longs delays between opening an issue or pull request and a first response from a maintainer can be demotivating.
+Similarly, after waiting a long time for a response, the contributor may no longer have time to follow up, having moved on to other tasks and projects.
+
+Make sure that you have [configured notifications for your project](./collaborators_team.md#staying-in-the-loop) so that you will know about it when somebody gets involved.
```

Comment:
```suggestion
Make sure that you have [configured notifications for your project](./collaborating_team.md#staying-in-the-loop) so that you will know about it when somebody gets involved.
```